content/blogs/2024-01-16-gorgias.md

```diff
@@ -0,0 +1,122 @@
+
+## The Control Plane: Data Orchestrator
+
+While internal data analysis often doesn’t necessitate real-time processing, batch processing is optimally managed through orchestration tools, offering the capability to trigger and schedule tasks, much like a cron job.
+
+Yet, some are tempted to consolidate all logic within highly customizable systems like Airflow. Doing ETL, transformation and reverse ETL all with custom code brings some drawbacks:
+
+- Orchestration logic involving complex decorators can present a steep learning curve.
+- Intricacies of modularity and deployment strategies, such as building Docker images and creating PyPI packages, add complexity to an already sophisticated process.
+- Impact on the entire stack is not always straightforward when implementing changes, especially when updating shared components.
+
+Several months ago, Gorgias faced the limitations of embedded schedulers leading them to the adoption of a dedicated data orchestration tool.
+
+Although many modern data tools integrate schedulers (Hightouch, Dbt cloud, Airbyte, GCP with Cloud Scheduler and scheduled queries), they are not build for handling complex orchestration needs, such as retry policies, event-triggered workflows, and intricate dependencies.
+
+Also, managing numerous execution logs across different tabs further underscored the need for a centralized orchestrator — a main control plane to streamline operations and enhance visibility.
+
+
+## Declarative Data Orchestration with Kestra
+
+Kestra offers an innovative way to deal with data pipelines based on YAML definition. Fulfilling their need for Infrastructure As Code pattern.
+Other important aspects :
+
+- Native integration with all the tools Gorgias is already using: Airbyte, BigQuery, PostgreSQL databases, dbt (CLI and cloud), GitHub, Slack, Hightouch, etc.
+- A rich user interface to define tasks and manage flow executions
+- Natively support Webhook trigger
+- Terraform provider to allows versioning and modularity on top of Flows and thus introduces the best software engineering practices.
+- Designed to scale and handle millions of flows in parallel, scaling it is made easy out of the box.
+- Full control of data (self-hosted on Kubernetes)
+- Easy to contribute to plugins.
+
+With Kestra, the Gorgias team is addressing the following needs:
+
+- Triggering Airbyte sync followed by dbt transformations
+- Have proper retry policies. Especially for dbt jobs, as they moved to capacity-based BigQuery pricing they can end up with resource exhaustion issues more often.
+- Scheduling Hightouch sync at the end of dbt jobs for better freshness
+- Triggering ETL python scripts with easy monitoring based on Webhook events
+- Centralized place for all their logs
+- Integrated UI allows you to define on-the-fly flows using a nice workflow editor and provide a large collection of blueprints :
```

Comment:
```suggestion
Kestra is transforming data pipeline management by using a declarative approach, making it easier to bring Infrastructure as Code best practices to data engineering workflows. That declarative interface fits seamlessly into Gorgias' existing toolkit, which includes Airbyte, BigQuery, PostgreSQL, dbt, GitHub, Slack, Hightouch, and others. With Kestra, creating and overseeing tasks becomes more intuitive thanks to its comprehensive UI. Kestra also supports Webhook triggers natively and integrates with Terraform to add versioning and modularity to data flows. 

One of Kestra's strengths is its ability to scale effortlessly. It can handle millions of flows simultaneously and provides users full control over their data, including the option for self-hosting on Kubernetes. Additionally, contributing to Kestra's plugin ecosystem has been a straightforward process.

For the Gorgias team, streamlining processes like initiating Airbyte synchronization followed by dbt transformations and setting up efficient retry policies for dbt jobs has been particularly valuable, as they have shifted to BigQuery's capacity-based pricing. Kestra also enables the Gorgias team to schedule Hightouch synchronization after dbt jobs to ensure data remains current, trigger ETL Python scripts with easy monitoring, centralize their log management, and use an integrated UI. This UI allows for the quick creation of flows with a user-friendly workflow editor and offers a wide range of blueprints for various needs.
```

too many bullet points already in the previous section, I'd recommend rewriting entirely as above